superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeGallery.tsx

```diff
@@ -332,6 +333,38 @@ const thumbnailContainerCss = (theme: SupersetTheme) => css`
   }
 `;
 
+const BetaBadge = styled.div`
+  ${({ theme }) => `
+    border: 1px solid #1985a0;
+    box-sizing: border-box;
+    border-radius: 4px;
+    background: #ffffff;
+    line-height: ${theme.gridUnit * 2.5}px;
+    color: #1985a0;
+    font-size: 12px;
```

Comment:
```suggestion
    font-size: ${theme.gridUnit * 3}px;
```

Reply:
Here may prefer to use `theme.typography.sizes.s`